start.js

```diff
@@ -23,6 +25,8 @@ function start (opts) {
   const sapn = opts['shared-access-policy-name'] || process.env.PINO_SHARED_ACCESS_POLICY_NAME
   const sapk = opts['shared-access-policy-key'] || process.env.PINO_SHARED_ACCESS_POLICY_KEY
   const sas = opts['sas'] || process.env.PINO_SHARED_ACCESS_SIGNATURE
+  const max = opts['max'] || socketCount
+  const agent = new https.Agent({keepAlive: true, maxSockets: max})
```

Comment:
yes, been thinking about this more and need to do some refactoring.  Thanks for the feedback.